src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/MessageListItem.vue

```diff
@@ -21,7 +21,7 @@
 
 <script lang="ts">
 import { defineComponent, type PropType } from "vue";
-
+import Callout from "@/components/Callout.vue";
```

Comment:
I don't think we need the Callout in this component. I don't see it referenced in the component.